TypeCobol.Analysis/Dfa/ValueOrigin.cs

```diff
@@ -137,6 +136,30 @@ void ProcessSetStatement(SetStatement set)
                             break;
                     }
                 }
+
+                void ProcessDataDefinition(DataDefinition dataDef)
+                {
+                    VariableSymbol vs = (VariableSymbol)dataDef.SemanticData;
+                    if (vs.Value != null)
+                    { //(2.1.1.3.1) Take the initial value and add it to the origins path.
+                        if (vs.Value is Value value)
+                        {                            
```

Comment:
I think we can safely assume that dataDef.SemanticData is a non-null VariableSymbol with a non-null Value at that point. No need to test it in an `if`, a `Debug.Assert` is more appropriate.

Reply:
Hum... this is not for testing dataDef.SemanticData but the initial value.

Reply:
Yes, I know, what I say is that you can call `(Value) ((VariableSymbol) dataDef.SemanticData).Value` without any prior check and get a non-null value because the DataDefinition we'll get here have been previously collected with that criterion. So an Assertion is enough, no need for more `if` testing.

Reply:
Done in e9f226e